mergify_engine/clients/http.py

```diff
@@ -206,7 +206,7 @@ def __init__(
         headers: typing.Optional[httpx_types.HeaderTypes] = None,
         timeout: httpx_types.TimeoutTypes = DEFAULT_TIMEOUT,
         base_url: httpx_types.URLTypes = "",
-    ):
+    ) -> None:
```

Comment:
could be in a different pr but ok :)

Reply:
I don't get because mypy didn't see it before...